models/us_fdic_bankfind/code/validate.py

```
@@ -0,0 +1,202 @@
+             max(if(indicator_id = 'DEP', value, null)) dep,
+             max(if(indicator_id = 'NETINC', value, null)) netinc
+      from `{PROJECT}.{DATASET}.financials_indicator`
+      where year = 2026 and quarter = 2
+      group by 1, 2, 3
+    )
+    select count(*) n,
+           countif(w.total_assets != l.asset) bad_assets,
+           countif(w.total_deposits != l.dep) bad_deposits,
+           countif(w.net_income != l.netinc) bad_income
+    from w join l using (year, quarter, cert)
+    """
```

Ruff (0.16.2)

[error] Possible SQL injection vector through string-based query construction

(S608)